### PR TITLE
Decode morse

### PR DESCRIPTION
**Pull request by Esmatullah12**
In this project, we worked on the following tasks
- Run a program using the command line.
- Use Ruby syntax for basic programming operations.
- Use Ruby to solve basic coding challenges.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,191 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (shella12) [inline README.md:191]: - [ ] The link to your `MIT LICENCE` is broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @Esmatullah12 and @Kingno3l , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Message was decoded accurately ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kingno3l** [comment]
Okay on it!!!

**Comment by Kingno3l** [comment]
Done!!!

**Review by TadesseDev — APPROVED** [review]
### Hi @Esmatullah12 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅


## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**